client/app/src/main/java/com/example/criengine/Adapters/BorrowerBooksListAdapter.java

```diff
@@ -82,8 +106,8 @@ public void onClick(View v) {
 
         statusText.setText(book.getStatus());
 
-        actionButton.setEnabled(true);
         actionButton.setVisibility(View.VISIBLE);
+        actionButton.setEnabled(true);
         switch (book.getStatus()) {
             case "borrowed":
 //            statusText.setTextColor(view.getResources().getColor(R.color.status_accepted));
```

Comment:
We might be able to just remove this actually, i don't think we're worrying about coloring for the text. Lets us clean up the file a tiny bit :D